gpytorch/mlls/deep_predictive_log_likelihood.py

```diff
@@ -0,0 +1,36 @@
+from ..models.deep_gps.dspp import DSPP
+from ._approximate_mll import _ApproximateMarginalLogLikelihood
+
+
+class DeepPredictiveLogLikelihood(_ApproximateMarginalLogLikelihood):
+    """
+    A wrapper to make a GPyTorch approximate marginal log likelihoods compatible with Deep GPs.
```

Comment:
nit: DSPPs not Deep GPs.

Reply:
Also very nit: could you do "compatible with :obj:`~gpytorch.models.deep_gps.DSPP`?